pkg/ostree/app.js

```diff
@@ -0,0 +1,328 @@
+define([
+    "jquery",
+    "base1/cockpit",
+    "base1/angular",
+    "base1/moment",
+    "ostree/client",
+], function($, cockpit, angular, moment, client) {
+    'use strict';
+
+    var _ = cockpit.gettext;
+    var phantom_checkpoint = phantom_checkpoint || function () { };
+
+    function track(item) {
```

Comment:
This function has a name that makes me think it does something different than what it does.